records/generator_test.go

```diff
@@ -308,7 +316,7 @@ func TestInsertState(t *testing.T) {
 	}
 
 	// test for 5 A names
-	if got, want := len(rg.As), 16; got != want {
+	if got, want := len(rg.As), 18; got != want {
```

Comment:
Not that we should change this right now, but this test is not good. It has to change every time we add or remove a case. Better to test for the individual records.